lib/themes/dosomething/paraneue_dosomething/js/campaign/HotModule.js

```diff
@@ -1,3 +1,13 @@
+/* eslint-disable */
```

Comment:
Nope! I did add the disable flag to a lot of files to save some time upfront, but the idea is that we can just remove that flag as soon as we start modifying a file. The new linting rules will enforce use of `let` and `const` instead of `var` & disallow old AMD-style requires, but otherwise it's same-old same-old.